### PR TITLE
Fix vercel redeploy

### PR DESCRIPTION
**Pull request by jprochazk**
### What
* Closes https://github.com/rerun-io/rerun/issues/4091

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested [demo.rerun.io](https://demo.rerun.io/pr/4119) (if applicable)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG

- [PR Build Summary](https://build.rerun.io/pr/4119)
- [Docs preview](https://rerun.io/preview/2fb8dc91fad2ed71a93686e1e283031d224ef63e/docs) 
- [Examples preview](https://rerun.io/preview/2fb8dc91fad2ed71a93686e1e283031d224ef63e/examples) 
- [Recent benchmark results](https://ref.rerun.io/dev/bench/)
- [Wasm size tracking](https://ref.rerun.io/dev/sizes/)

### CODE REVIEW

**Comment by jprochazk** [comment]
Now it's clear what the problem was: https://github.com/rerun-io/rerun/actions/runs/6721155529/job/18266251435#step:3:13

**Comment by jprochazk** [comment]
Seems to work after giving it a fresh token

File: .github/actions/deploy-vercel/index.mjs [hunk 1]
```diff
@@ -11,25 +11,35 @@ const releaseCommit = getRequiredInput("release_commit");
...
-console.log(`Fetching \`RELEASE_COMMIT\` env var`);
-const env = await client.envs(team.id, project.id).then(find("key", "RELEASE_COMMIT"));
-assert(env, `failed to get \`RELEASE_COMMIT\` env var`);
-
-console.log(`Setting \`RELEASE_COMMIT\` env to \`${releaseCommit}\``);
-await client.setEnv(team.id, project.id, env.id, { key: "RELEASE_COMMIT", value: releaseCommit });
-
-console.log(`Triggering redeploy`);
-await client.redeploy(team.id, deployment.uid, "landing");
+info`Fetching team "${teamName}"`;
```
Comment (emilk) [inline .github/actions/deploy-vercel/index.mjs:14]: What magic syntax is this O.o

**Review by emilk — APPROVED** [review]
Thanks for improving the error reporting around this!

File: .github/actions/deploy-vercel/index.mjs (hunk 1 above)
Comment (jprochazk) [reply .github/actions/deploy-vercel/index.mjs:14]: [Tagged template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates)

**Comment by jprochazk** [comment]
One last thing: Added the ability to run the deployment manually given all the same inputs as the CI, via `manual.mjs` (making things work local-first 🙂)

**Pull request merged**